docs/UserGuide.md

```diff
@@ -363,7 +397,7 @@ Format: `C delete INDEX`
 
 Examples: `C delete 1`
 
-Remark: The `INDEX` provided must be positive and cannot be larger than the number of Contacts.
+Remark: The `INDEX` provided must be positive and  within the range of contacts in the current window.
```

Comment:
The `INDEX` provided must be one of the contact' indexes being shown on the window? Does is sound better?